### ISSUE TITLE
Issue installing python deps

### ISSUE THREAD

**Issue by mikedotexe** [matnad/pollmaster#90]
Maybe this is an issue on Mac, but following the directions in `setup.md` I ran:

```
pip install -r requirements.txt
```

(where `which pip` is where I'd kinda of expect it to be: `/Users/mike/opt/miniconda3/envs/pollmaster/bin/pip`)

I consistently get a version issue:

```
ERROR: Cannot install -r requirements.txt (line 9) and aiohttp==3.7.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested aiohttp==3.7.2
    dblpy 0.3.4 depends on aiohttp<3.7.0 and >=3.3.0
```

Looks like the requirements file hasn't been modified in a while, which is surprising.

**Comment by boonebytes** [comment]
Nope... that's not a Mac-specific thing. I set up my "development environment" in Docker (see pull-request #89 here for what I've done so far).

The requirements.txt file has a few mismatching dependencies. Some of the suggestions on the support Discord (DEV channel) is to remove the version specifics from each line that throws the error. Alternately, you can look at the docker-requirements.txt file in my pull request.

You might also need a specific version of Python; newer versions of Python could be incompatible; I'm using Python 3.7.

Hope it helps :)
